pytest_reportportal/service.py

```diff
@@ -231,6 +266,79 @@ def _stop_if_necessary(self):
         except queue.Empty:
             pass
 
+
+    @staticmethod
+    def _add_item_hier_parts_dirs(item, hier_flag, dirs_level, report_parts, dirs_parts, rp_name=""):
```

Comment:
we do not use rp_name parameter in method

Reply:
I use `rp_name` to accumulate string for item name. And it depends on should we report hierarchy level or not.
The idea is that we should be able to identify what test item was reported (by path) and to do not report few times path parts.

Example
dir1/dir2/module1.py::TestClass1::test1 [param1=1]
dir1/dir2/module1.py::TestClass1::test1 [param1=2]
dir1/dir2/module1.py::TestClass1::test2

for: hier_dir =1, hier_module=1, hier_class=1, hier_param=0
we report: dir1 -> dir2 -> module1.py -> TestClass1 -> test1 [param1=1]
                                                                                 -> test1 [param1=2]
                                                                                 -> test2

for: hier_dir =0, hier_module=1, hier_class=1, hier_param=0
we report: dir1/dir2/module1.py -> TestClass1 -> test1 [param1=1]
                                                                        -> test1 [param1=2]
                                                                        -> test2

for: hier_dir =1, hier_module=0, hier_class=0, hier_param=0
we report: dir1 -> dir2 -> module1.py::TestClass1::test1 [param1=1]
                                    -> module1.py::TestClass1::test1 [param1=2]
                                    -> module1.py::TestClass1::test2


for: hier_dir =1, hier_module=0, hier_class=0, hier_param=1
we report: dir1 -> dir2 -> module1.py::TestClass1::test1 -> test1 [param1=1]
                                                                                      -> test1 [param1=2]
                                    -> module1.py::TestClass1::test2

Reply:
I still can't find any usages of value from rp_name parameter in _add_item_hier_parts_dirs method.
You assign new value to rp_name variable on line 289 and return it on line 291.

Reply:
I added `rp_name` to this func to be consistent with other `_add_item_hier_...` functions. 
Since directory was first stage we don't actually used it. 
But now I fixed it and you can pass some real value to this func and it will be added to item name.